home/vlt-adm/system/register_vulture_repos.sh

```diff
@@ -1,11 +1,13 @@
 #!/usr/bin/env sh
 # Usage update_repositories [prefix_dir]
 
+OS_RELEASE=`/usr/bin/grep 'branch="' $1/etc/hbsd-update.conf | /usr/bin/sed 's/branch="\(.*\)"/\1/'`
+OS_BRANCH_VERSION=`/usr/bin/grep 'os_version="' $1/etc/hbsd-update.conf | /usr/bin/sed 's/os_version="\(.*\)"/\1/'`
 pkg_url="http://pkg.vultureproject.org/"
 vulture_conf="Vulture.conf"
 pkg_ca="pkg.vultureproject.org"
-update_url="http://updates.vultureproject.org/"
-vulture_update_conf="hbsd-update.conf"
+update_url="http://hbsd.vultureproject.org/"
+vulture_update_conf="hbsd-update-${OS_RELEASE}-${OS_BRANCH_VERSION}.conf"
```

Comment:
move the dynamic fields in the `update_repositories()` and control their value and set to default if necessary